README.md

```diff
@@ -26,5 +26,8 @@ Any questions let us know
+<td align="center"><p><a href="https://github.com/RemcoHalman">Remco Halman</a></p><img src="https://user-images.githubusercontent.com/25010775/91281626-9cc5d200-e788-11ea-843f-52a10890b485.jpg" /><p><a href="https://github.com/EddieJaoudeCommunity/awesome-github-profiles/issues/69">(:100: give your vote)</a></p></td>
+<td align="center"><p><a href="https://github.com/w3bdesign">Daniel F</a></p><img src="(https://user-images.githubusercontent.com/45217974/91754997-ae650a80-ebca-11ea-97fd-dfa011b49d79.png" /><p><a href="https://github.com/EddieJaoudeCommunity/awesome-github-profiles/issues/72">(:100: give your vote)</a></p></td>
+</tr></table>
```

Comment:
Thank you for the contribution, these changes are autogenerated by the GitHub Action. I do not think it is a blocker for the PR, as the GitHub Action will overwrite this.

Lets try it and find out. The `json` file are the important changes

Reply:
Yes the GitHub Action fixed this, thank you for testing 🤓